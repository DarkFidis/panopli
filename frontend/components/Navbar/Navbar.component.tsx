import React from "react";
import styles from '../../styles/Navbar.module.css'

export const Navbar: React.FC = () => (
  <nav className={styles.navbar}>
    <ul>
      <li><a data-scroll="home" href="#home" className="active">home</a></li>
      <li><a data-scroll="feature" href="#feature">feature</a></li>
      <li><a data-scroll="about" href="#about">about</a></li>
      <li><a data-scroll="gallery" href="#gallery">gallery</a></li>
      <li><a data-scroll="review" href="#review">review</a></li>
      <li><a data-scroll="contact" href="#contact">contact</a></li>
    </ul>
  </nav>
)

export default Navbar
