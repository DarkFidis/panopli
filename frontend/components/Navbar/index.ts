import NavbarComponent, { Navbar } from "./Navbar.component";
export { Navbar }
export default NavbarComponent
