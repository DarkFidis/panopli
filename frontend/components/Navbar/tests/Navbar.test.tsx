import * as React from 'react'

import { render } from "@testing-library/react";
import Navbar from "../index";

describe('Navbar component unit test', () => {
  it('should match snapshot', () => {
    const { container } = render(<Navbar />)
    expect(container).toMatchSnapshot()
  })
})
