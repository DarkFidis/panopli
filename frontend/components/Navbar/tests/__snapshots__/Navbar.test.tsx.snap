// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component unit test should match snapshot 1`] = `
<div>
  <nav
    class="navbar"
  >
    <ul>
      <li>
        <a
          class="active"
          href="#home"
        >
          accueil
        </a>
      </li>
      <li>
        <a
          href="#feature"
        >
          restaurants
        </a>
      </li>
      <li>
        <a
          href="#about"
        >
          a propos
        </a>
      </li>
      <li>
        <a
          href="#review"
        >
          avis
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
